util/missingcrypto.txt

```diff
@@ -1274,6 +1274,9 @@ WHIRLPOOL_BitUpdate(3)
 WHIRLPOOL_Final(3)
 WHIRLPOOL_Init(3)
 WHIRLPOOL_Update(3)
+WPACKET(3)
+WPACKET_init_der(3)
+WPACKET_init_null_der(3)
```

Comment:
Side note: I really think missing internal function documentation should not go into missingcrypto.txt. There could be a different file for those - but I like the concept that missingcrypto.txt never gets additions. I know there be some a few occasions where we have allowed it  - but this should be the general principle.

Reply:
PR welcome, and specially one that documents PACKET / WPACKET :wink:
(c'mon, you must have seen that coming!)

Reply:
I've taken it upon myself to adjust find-doc-nits, FYI